nvflare/apis/utils/fl_context_utils.py

```diff
@@ -20,17 +20,19 @@
 from nvflare.fuel.sec.audit import AuditService
 from nvflare.fuel.utils import fobs
 
+logger = logging.getLogger("fl_context_utils")
+
 
 def get_serializable_data(fl_ctx: FLContext):
-    logger = logging.getLogger("fl_context_utils")
     new_fl_ctx = FLContext()
     for k, v in fl_ctx.props.items():
         if k not in NonSerializableKeys.KEYS:
             try:
                 fobs.dumps(v)
                 new_fl_ctx.props[k] = v
             except BaseException as e:
-                logger.warning(generate_log_message(fl_ctx, f"Object is not serializable (discarded): {k} - {v}: {e}"))
+                logger.warning(generate_log_message(fl_ctx, "Object is not serializable (discarded): "
+                                                            f"{k} - {type(v)}: {e}"))
```

Comment:
Should we make this more detailed?

"Key {k} in FLContext of type {type(v)} is not serializable (discarded): {e}"

Reply:
Changed.